src/tokens/tokens.service.ts

```diff
@@ -63,13 +63,13 @@ export class TokensService {
     this.shortPrefix = shortPrefix;
   }
 
-  private postOptions(requestId?: string) {
+  private postOptions(operator?: string, requestId?: string) {
     const from = `${this.shortPrefix}-from`;
     const sync = `${this.shortPrefix}-sync`;
     const id = `${this.shortPrefix}-id`;
     return {
       params: {
-        [from]: this.identity,
+        [from]: operator ?? this.identity,
```

Comment:
Should we even have a default in the environment, or should we move toward requiring `operator` on every call?

Reply:
My gut says provide the operator on every call.